### PR TITLE
[release/3.2] Generate changelog for 3.2.0

### PR DESCRIPTION
**Pull request by sergepetrenko**
Also, remote unreleased/ entries and add missing downgrade versions to the downgrade versions list.

NO_DOC=changelog
NO_TEST=changelog
NO_CHANGELOG=changelog

### CODE REVIEW

File: changelogs/3.2.0.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+* Introduced the `tarantool.build.tzdata_version` option to get
+  the `tzdata` version.
+
+#### Popen
+
+* Eliminated polling in `<popen handle>:wait()`, so now it reacts to SIGCHLD
+  faster and performs less unnecessary work (gh-4915).
+* Added the `timeout` parameter for `<popen handle>:wait()` (gh-4916).
+
+### Box
```
Comment (Totktonada) [inline changelogs/3.2.0.md:82]: Merge with 'Core'?

File: changelogs/3.2.0.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+### Lua
+
+#### Popen
+
+* Defined the behavior of in-progress `<popen handle>:wait()` when
+  `<popen handle>:close()` is called in another fiber: now it returns an error
+  instead of accessing a freed memory and, likely, hanging forever (gh-7653).
+
+### Box
```
Comment (Totktonada) [inline changelogs/3.2.0.md:138]: Rename to 'Core'?

**Review by Totktonada — APPROVED** [review]
I've no objections.

File: changelogs/3.2.0.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  spaces. For user spaces, this field will always mirror the `is_sync` option
+  set for the space (gh-9723).
+
+### Lua
+
+* Added support for the `level` argument in `box.error` and `box.error.new`
+  when an error is created using an error code (gh-9792).
+* Added `box.error.is` that checks whether its argument is a `box.error`.
+* Introduced a standard Lua way to get the length of `key_def` (gh-10111).
+* Introduced Lua implementation of protobuf encoder (gh-9844).
```
Comment (lenkis) [inline changelogs/3.2.0.md:66]:
```suggestion
* Introduced Lua implementation of the protobuf encoder (gh-9844).
```

File: changelogs/3.2.0.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+* Deprecated using `cdata` values with `metrics` module `histogram:observe`.
```
Comment (lenkis) [inline changelogs/3.2.0.md:67]:
```suggestion
* Deprecated using `cdata` values with `histogram:observe` from the `metrics` module.
```

File: changelogs/3.2.0.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+* Updated memtx metrics descriptions from `metrics` module to be consistent.
```
Comment (lenkis) [inline changelogs/3.2.0.md:68]:
```suggestion
* Updated memtx metrics descriptions from the `metrics` module to be consistent.
```

File: changelogs/3.2.0.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+* Added new metrics to `metrics` module: `tnt_memtx_tuples_data_total`,
```
Comment (lenkis) [inline changelogs/3.2.0.md:69]:
```suggestion
* Added new metrics to the `metrics` module: `tnt_memtx_tuples_data_total`,
```

File: changelogs/3.2.0.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+
+* Eliminated polling in `<popen handle>:wait()`, so now it reacts to SIGCHLD
+  faster and performs less unnecessary work (gh-4915).
+* Added the `timeout` parameter for `<popen handle>:wait()` (gh-4916).
+
+### Box
+
+* Added the source tuple as a second argument for the functional variant of the
+  default field values (gh-9825).
+* Added the `tuple:format` method to get a format of a tuple (gh-10005).
```
Comment (lenkis) [inline changelogs/3.2.0.md:86]:
```suggestion
* Added the `tuple:format` method to get the tuple format (gh-10005).
```

**Review by lenkis — APPROVED** [review]

File: changelogs/3.2.0.md (hunk 2 above)
Comment (sergepetrenko) [reply changelogs/3.2.0.md]: Done, thanks!

**Pull request merged**